cli/popper/commands/cmd_archive.py

```diff
@@ -161,10 +183,7 @@ def publish_snapshot(publish_url, params):
             "{} and the DOI URL {}".format(doi, doi_url)
         )
     else:
-        pu.fail(
-            "Status {}: Could not publish the snapshot. "
-            "Try again later.".format(published.status_code)
-        )
+        pu.fail("Could not publish the snapshot. Try again later.")
```

Comment:
The previous messaged was more informative. Showing the status might help users understand what's wrong with their request.